include/libxal.h

```diff
@@ -95,6 +95,36 @@ xal_inode_pp(struct xal_inode *inode);
  */
 struct xal;
 
+/**
+ * Returns the extent information in bytes.
+ * 
+ * Assumes that the extent information has not already been converted using xal_extent_in_bytes()
+ * or xal_extent_in_lba().
+ * 
+ * @param xal The xal struct obtained when opened with xal_open()
+ * @param extent A pointer to a non-converted `xal_extent` 
+ * @param output A pointer to the output of the convertion
+ * 
+ * @return On success a 0 is returned. On error, negative errno is returned to indicate the error.
+ */
+int
+xal_extent_in_bytes(struct xal *xal, const struct xal_extent *extent, struct xal_extent *output);
```

Comment:
It seems here that the input is `const` instead of the output as the commit message reads.

> The return value of the converter functions are `const`